_sass/_navigation.scss

```diff
@@ -84,16 +72,16 @@ ul, ul ul {
 .navContainer {
 	font-weight:bold;
 	margin-top:2px;
-	margin-left:.6em;
+	margin-left:.3em;
```

Comment:
shifting nav menu to accomodate carets (same with other margin-left changes below)